### PR TITLE
app-arch/lxqt-archiver: add version 0.1.0 (new package)

### PR DESCRIPTION
**Pull request by Chiitoo**

### CODE REVIEW

File: app-arch/lxqt-archiver/lxqt-archiver-0.1.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+BDEPEND="
+	dev-qt/linguist-tools:5
+	>=dev-util/lxqt-build-tools-0.6.0
+	virtual/pkgconfig
+"
+DEPEND="
+	dev-libs/glib:2
+	dev-libs/json-glib
+	dev-qt/qtcore:5
+	dev-qt/qtdbus:5
```
Comment (a17r) [inline app-arch/lxqt-archiver/lxqt-archiver-0.1.0.ebuild:31]: I didn't find linking to that, can you check?

Comment (Chiitoo) [reply app-arch/lxqt-archiver/lxqt-archiver-0.1.0.ebuild:31]: I was wondering about that one as well, and even tested building and running without it, which worked as far as I tested.

It was already a requirement in the "initial import" in the repo, so I figured I'd ask them about it.  Didn't get to it quite yet though.

**Comment by Chiitoo** [comment]
> Found a shell script in lib64 install path, are they serious?
> 
> `/usr/lib64/lxqt-archiver/isoinfo.sh`

I guess something like '/usr/libexec/lxqt-archiver/isoinfo.sh' would be better?  (That's where 'engrampa' seems to install said file).

**Comment by Chiitoo** [comment]
I think I'll add this with the LXQt 0.15.0 release (or well, the 0.1.1 version that will have these two issues fixed).

Thanks!

**Pull request closed without merging**